### ISSUE TITLE
Not an issue, really a suggestion.

### ISSUE THREAD

**Issue by welzie** [jpgfour/reading-log#1]
https://github.com/jpgfour/reading-log/blob/e3a1b4577f8a67ac0d3f9c071711f950c8ae45df/src/main/resources/templates/add.html#L18

How do you feel about using `console.log("your message");` and the js console in developer tools to see these messages? Firefox and chrome both have a js console https://developers.google.com/web/tools/chrome-devtools/console/javascript.

**Comment by jpgfour** [comment]
Oh, instead of the alerts?

**Comment by welzie** [comment]
Yes. Alerts are fine, but they force you to have to click each one and that
can be annoying.
